src/components/SuggestedJourneys/SuggestedJourneys.tsx

```diff
@@ -23,17 +24,39 @@ function getTimeOfDayClass(params: JourneyParams) {
     } else return styles.evening;
 }
 
+function getTimeofDay(params: JourneyParams) {
+    const { time } = params;
+    if (time === undefined) {
+        return "Midday";
+    } else if (time >= 18000 && time <= 39600) {
+        return "Morning";
+    } else if (time > 39600 && time <= 61200) {
+        return "Midday";
+    } else return "Evening";
+}
+
 const SuggestedJourneys = (props: Props) => {
     const { journeys } = props;
     const divArr = [] as any;
     for (let i = 0; i < journeys.length; ++i) {
+        const journey = journeys[i];
         divArr.push(
             <div className={styles.journey} key={i}>
-                <div className={getTimeOfDayClass(journeys[i])}>
-                    <a href={getJourneyUrl(journeys[i])}>
-                        <h2>Journey Option {i + 1}:</h2>
-                        <p>Depart from: {stationsById[journeys[i]["fromStationId"]].name}</p>
-                        <p>Arrive at: {stationsById[journeys[i]["toStationId"]].name}</p>
+                <div className={getTimeOfDayClass(journey)}>
+                    <a href={getJourneyUrl(journey)}>
+                        <h4>
+                            {" "}
```

Comment:
This space may not be intentional! Spaces inside JSX (React) elements can count for actual spaces inserted into the document, affecting the layout — Prettier tries to make these more explicit by writing them out as an inserted string (`{" "}`). All in all, you likely want to remove this.